test/purchases.test.js

```diff
@@ -50,21 +50,11 @@ describe('POST /albums/:id', () => {
         });
     });
 
-    test('Send a null token', () => {
-      albumsListMock(1, albumTitle);
+    test.each([{}, { token }])('Send null and invalid token', body => {
```

Comment:
I like this, but beware that you'll end up with two test of the same name, which can be confusing if one fails and the other passes. Try adding something like `Send null or invalid token in the following JSON: %j` to help you differentiate between them. You can do this in the next PR.